src/App.vue

```diff
@@ -74,7 +74,7 @@
 
     <!-- Initial Page Load Transition -->
     <transition name="fade">
-      <div class="loading-container" v-show="!haveData">
+      <div class="loading-container" v-show="!haveData && !isErrorDialog">
```

Comment:
If there is a data fetch error then we don't set haveData (so that the pages aren't rendered). But we need to hide the spinner to see any error dialogs.